autofit/config/output.yaml

```diff
@@ -0,0 +1 @@
+default: true
```

Comment:
Add description to config file, like here:

https://github.com/Jammy2211/autofit_workspace/blob/release/config/notation.yaml

Add comments to each entry like here:

https://github.com/Jammy2211/autofit_workspace/blob/release/config/general.yaml